Speech/src/Connection/ConnectionInterface.php

```diff
@@ -21,8 +21,7 @@
  * Represents a connection to
  * [Google Cloud Speech](https://cloud.google.com/speech/).
  *
- * @deprecated This class is no longer supported and will be removed in a future
- * release.
+ * @internal
```

Comment:
We should leave these as `@deprecated`! You can still add `@internal` if you'd like, but we should keep the `@deprecated` tag as well.